comparison/loadtest.go

```diff
@@ -220,21 +293,14 @@ func initLoadTest(t *terraform.Terraform, buildCfg BuildConfig, dumpFilename str
 	}
 	loadDBDumpCmd.Value = dbCmd
 
-	resetBucketCmds := []localCmd{}
-	if s3BucketURI != "" && tfOutput.HasS3Bucket() {
-		deleteBucketCmd := localCmd{
-			msg:   "Emptying S3 bucket",
-			value: []string{"--profile", t.Config().AWSProfile, "s3", "rm", "s3://" + tfOutput.S3Bucket.Id, "--recursive"},
-		}
-
-		prepopulateBucketCmd := localCmd{
-			msg:   "Pre-populating S3 bucket",
-			value: []string{"--profile", t.Config().AWSProfile, "s3", "cp", s3BucketURI, "s3://" + tfOutput.S3Bucket.Id, "--recursive"},
-		}
-
-		resetBucketCmds = []localCmd{deleteBucketCmd, prepopulateBucketCmd}
+	cfg, err := config.LoadDefaultConfig(context.Background(), config.WithSharedConfigProfile(t.Config().AWSProfile))
+	if err != nil {
+		return fmt.Errorf("error loading aws config: %w", err)
 	}
 
+	s3client := s3.NewFromConfig(cfg)
+	bucket := s3ClientWrapper{S3Client: s3client}
```

Comment:
The name of the variable is still a bit confusing. I was thinking of something like:
```suggestion
	s3client := s3ClientWrapper{S3Client: s3.NewFromConfig(cfg)}
```